build/cqs.tsv

```diff
@@ -170,47 +171,46 @@ org.calrissian.mango:mango-core	1.2.0	compile
+org.eclipse.emf:org.eclipse.emf.common	2.15.0	compile
+org.eclipse.emf:org.eclipse.emf.ecore	2.15.0	compile
+org.eclipse.emf:org.eclipse.emf.ecore.xmi	2.15.0	compile
 org.eclipse.xsd:org.eclipse.xsd	2.12.0	compile
 org.ejml:ejml-core	0.34	compile
 org.ejml:ejml-ddense	0.34	compile
 org.gdal:gdal	2.0.0	compile
-org.geotools:gt-api	20.0	compile
-org.geotools:gt-coverage	20.0	compile
-org.geotools:gt-cql	20.0	compile
-org.geotools:gt-data	20.0	compile
-org.geotools:gt-epsg-hsql	20.0	compile
-org.geotools:gt-epsg-wkt	20.0	compile
-org.geotools:gt-geojson	20.0	compile
-org.geotools:gt-geotiff	20.0	compile
-org.geotools:gt-grid	20.0	compile
-org.geotools:gt-imageio-ext-gdal	20.0	compile
-org.geotools:gt-jdbc	20.0	compile
-org.geotools:gt-main	20.0	compile
-org.geotools:gt-metadata	20.0	compile
-org.geotools:gt-opengis	20.0	compile
-org.geotools:gt-process	20.0	compile
-org.geotools:gt-process-feature	20.0	compile
-org.geotools:gt-referencing	20.0	compile
-org.geotools:gt-render	20.0	compile
-org.geotools:gt-shapefile	20.0	compile
-org.geotools:gt-transform	20.0	compile
-org.geotools:gt-xml	20.0	compile
-org.geotools.jdbc:gt-jdbc-postgis	20.0	compile
-org.geotools.ogc:net.opengis.fes	20.0	compile
-org.geotools.ogc:net.opengis.ows	20.0	compile
-org.geotools.ogc:net.opengis.wfs	20.0	compile
-org.geotools.ogc:org.w3.xlink	20.0	compile
-org.geotools.xsd:gt-xsd-core	20.0	compile
-org.geotools.xsd:gt-xsd-fes	20.0	compile
-org.geotools.xsd:gt-xsd-filter	20.0	compile
-org.geotools.xsd:gt-xsd-gml2	20.0	compile
-org.geotools.xsd:gt-xsd-gml3	20.0	compile
-org.geotools.xsd:gt-xsd-ows	20.0	compile
-org.geotools.xsd:gt-xsd-wfs	20.0	compile
+org.geotools:gt-coverage	21.1	compile
+org.geotools:gt-cql	21.1	compile
+org.geotools:gt-epsg-hsql	21.1	compile
+org.geotools:gt-epsg-wkt	21.1	compile
+org.geotools:gt-geojson	21.1	compile
+org.geotools:gt-geotiff	21.1	compile
+org.geotools:gt-grid	21.1	compile
+org.geotools:gt-imageio-ext-gdal	21.1	compile
+org.geotools:gt-jdbc	21.1	compile
+org.geotools:gt-main	21.1	compile
+org.geotools:gt-metadata	21.1	compile
+org.geotools:gt-opengis	21.1	compile
+org.geotools:gt-process	21.1	compile
+org.geotools:gt-process-feature	21.1	compile
+org.geotools:gt-referencing	21.1	compile
+org.geotools:gt-render	21.1	compile
+org.geotools:gt-shapefile	21.1	compile
+org.geotools:gt-transform	21.1	compile
+org.geotools:gt-xml	21.1	compile
+org.geotools.jdbc:gt-jdbc-postgis	21.1	compile
+org.geotools.ogc:net.opengis.fes	21.1	compile
+org.geotools.ogc:net.opengis.ows	21.1	compile
+org.geotools.ogc:net.opengis.wfs	21.1	compile
+org.geotools.ogc:org.w3.xlink	21.1	compile
+org.geotools.xsd:gt-xsd-core	21.1	compile
+org.geotools.xsd:gt-xsd-fes	21.1	compile
+org.geotools.xsd:gt-xsd-filter	21.1	compile
+org.geotools.xsd:gt-xsd-gml2	21.1	compile
+org.geotools.xsd:gt-xsd-gml3	21.1	compile
+org.geotools.xsd:gt-xsd-ows	21.1	compile
+org.geotools.xsd:gt-xsd-wfs	21.1	compile
 org.hsqldb:hsqldb	2.3.0	compile
+org.huldra.math:bigint	0.7.1	compile
```

Comment:
pending... having some trouble tracking down the source code.

Reply:
@elahrvivaz what pulled this in!?

Reply:
Andrea Aime added it to jai-ext to speed up some operations I guess. I was able to find the source code by posting on their mailing list, so I'll have a cq up soon.